docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md

```diff
@@ -0,0 +1,72 @@
+# Connect to Nebula Graph
+
+Nebula Graph supports multiple types of clients, including a CLI client, a GUI client, and clients developed in popular programming languages. This topic provides an overview of Nebula Graph clients and a basic instruction on how to use the native CLI client, nebula-console.
+
+## Nebula Graph clients
+
+| Client                                                                 | Introduction                                           | Connection guide                                                                                                                                          |
+| ---------------------------------------------------------------------- | ------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [nebula-console](https://github.com/vesoft-inc/nebula-console)         | The native command-line interface of Nebula Graph      | [Connect to Nebula Graph with nebula-console](#use-nebula-console-to-connect-to-nebula-graph)                                                             |
+| [Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker) | The official graphical user interface for Nebula Graph | [Connect to Nebula Graph with Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker/blob/master/docs/nebula-graph-studio-user-guide-en.md) |
+| [nebula-go](https://github.com/vesoft-inc/nebula-go)                   | The official Golang client of Nebula Graph             | [nebula-go code example](https://github.com/vesoft-inc/nebula-go#usage-example)                                                                           |
+| [nebula-python](https://github.com/vesoft-inc/nebula-python)           | The official Python client of Nebula Graph             | [How to use nebula-python in your code](https://github.com/vesoft-inc/nebula-python#how-to-use-nebula-python-in-your-code)                                |
+| [nebula-java](https://github.com/vesoft-inc/nebula-java)               | The official Python client of Nebula Graph             | [Graph Client Example](https://github.com/vesoft-inc/nebula-java#graph-client-example)                                                                    |
+
+If you don't have a Nebula Graph database yet, we recommend that you try the cloud service. [Nebula Graph Cloud Service](https://www.nebula-cloud.io/) supports on-demand and deployment and fast building, and uses Nebula Graph Studio as its default client.
+
+## Use nebula-console to connect to Nebula Graph
+
+1. Download [the nebula-console binary file](TODO) [Coming soon].
+
+    ```bash
+    $ wget [Coming soon]
+    ```
+
+2. Use the binary file to connect to Nebula Graph.
+
+    ```bash
+    $ ./nebula-console [-addr IP] [-port port] -u <username> -p <password> [-t 120] [-e "an_nGQL_command" | -f filename.nGQL]
```

Comment:
what's the difference between [] and <>. I think [] means optional.

Reply:
Fair enough. addr and port should not be optional.